README.md

```diff
@@ -61,3 +61,15 @@ $ npm install
 ```shell
 $ npm run build
 ```
+
+## How to install
+
+### Please install pulsar-client in your project:
+
+```shell
+$ npm install pulsar-client
+```
+
+## Sample code
+
+Please refer to examples.
```

Comment:
There should be a link to the sample code here.

Reply:
Fixed readme.